subjects/devops/set-internal-vars/README.md

```diff
@@ -0,0 +1,32 @@
+## set-internal-vars
+
+### Instructions
+
+Create a script `set-internal-vars.sh`, which will allow you to create and print the following variables.
+
+- `MY_MESSAGE` which contains the string `"Hello World"`.
+- `MY_NUM` which contains the number `100`.
+- `MY_PI` which contains the number `3.142`.
+- `MY_ARR` which contains `(one two three four five)`
+
+Expected output:
+
+```console
+$ ./set-internal-vars.sh
+Hello World
+100
+3.142
+one, two, three, four, five
+$
+```
+
+### Hints
```

Comment:
no hints about how the student can manage the shell vars, you can mention "set, unset, ..."

Reply:
done